### PR TITLE
LA01_create_model_with_realtions

### PR DESCRIPTION
**Pull request by AliaksandrAnashkevich**

### CODE REVIEW

File: src/main/java/com/academia/library/model/Author.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+@Setter
+@EqualsAndHashCode
+@ToString
+@NoArgsConstructor
+@Entity
+@Table(name = "authors")
+public class Author {
+
+    @Id
+    @SequenceGenerator(name = "author_id_seq", sequenceName = "author_id_seq", allocationSize = 1, initialValue = 100)
```
Comment (DenisKiparizz) [inline src/main/java/com/academia/library/model/Author.java:29]: Lets do initialValue = 1 or try to avoid this

Comment (DenisKiparizz) [inline src/main/java/com/academia/library/model/Author.java:29]: I recommended you create new List/Set in entity. 
Please do that

File: src/main/java/com/academia/library/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne
+    @JoinColumn(name="tag_id", nullable=false)
+    private Tag tag;
+
+    @ManyToMany(mappedBy = "books")
+    private List<Order> orders;
+}
```
Comment (DenisKiparizz) [inline src/main/java/com/academia/library/model/Book.java:65]: As I recommended you latter please use commands CTRL+ALT+ L/O

File: src/main/java/com/academia/library/model/Authority.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Getter
+@Setter
+@NoArgsConstructor
```
Comment (DenisKiparizz) [inline src/main/java/com/academia/library/model/Authority.java:17]: You don't have to implement NoArgsConstructor without another constructor
Please repeat how constructor works in Java

**Review by DenisKiparizz — COMMENTED** [review]
Перед отправкой кода на ревью старайся делать self rewiew на код стаил и форматинг

File: src/main/java/com/academia/library/model/Author.java (hunk 1 above)
Comment (AliaksandrAnashkevich) [reply src/main/java/com/academia/library/model/Author.java:29]: Ok

Comment (AliaksandrAnashkevich) [reply src/main/java/com/academia/library/model/Author.java:29]: Ok

File: src/main/java/com/academia/library/model/Book.java (hunk 2 above)
Comment (AliaksandrAnashkevich) [reply src/main/java/com/academia/library/model/Book.java:65]: Ok

File: src/main/java/com/academia/library/model/Authority.java (hunk 3 above)
Comment (AliaksandrAnashkevich) [reply src/main/java/com/academia/library/model/Authority.java:17]: Ok

**Pull request closed without merging**